specification/_global/search/_types/profile.ts

```diff
@@ -75,7 +61,7 @@ export class AggregationProfileDelegateDebugFilter {
 }
 
 export class AggregationProfile {
-  breakdown: AggregationBreakdown
+  breakdown: Dictionary<string, long>
```

Comment:
I made it a generic dictionary since this is what it is on the ES server, and the field names evolve quite often.

Reply:
It'd be nice if we could ship the spec with an `AggregrationBreakdownKeys` that encodes known key constants into these dictionaries.